src/components/dashboard/Claim.js

```diff
@@ -131,62 +170,78 @@ const Claim = ({ navigation, screenProps, ...props }: ClaimProps) => {
   return (
     <Wrapper>
       <TopBar push={screenProps.push} />
-      <Section style={styles.mainContent}>
-        <Section.Text>GoodDollar allows you to collect</Section.Text>
-        <Section.Text>
-          <Text>G$s</Text>
-          <Text style={styles.everyDay}> every day</Text>
-        </Section.Text>
-      </Section>
-      <Section style={styles.nextIncome}>
-        <Section.Text>
-          {claimedToday.people} PEOPLE CLAIMED {claimedToday.amount} G$ TODAY!
-        </Section.Text>
+      <Section grow>
+        <Section.Stack grow={4} justifyContent="flex-start">
+          <Text>GoodDollar allows you to collect</Text>
+          <Section.Row justifyContent="center">
+            <Text fontFamily="slabBold" fontSize={36} color="#00c3ae">
```

Comment:
shouldn't this color come from the theme? Or at least be a constant in this file?